zerver/actions/message_edit.py

```diff
@@ -872,14 +877,15 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
                 )
             elif changed_messages_count == 1:
                 new_thread_notification_string = gettext_lazy(
-                    "A message was moved here from {old_location} by {user}."
+                    "{message_link} was moved here from {old_location} by {user}."
```

Comment:
This translation approach does not work. Read our guide on internationalization for developers:
https://zulip.readthedocs.io/en/latest/translating/internationalization.html

What you'll need to do is something like a manual variable for the URL that you substitute here, like `old_ location` and `user`, but have `[]()` syntax here.